nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/processor-details.jsp

```diff
@@ -103,6 +103,18 @@
                         <div class="clear"></div>
                     </div>
                     <div class="setting">
+                        <div id="details-execution-node-container" class="execution-node-setting">
```

Comment:
Same comment as above, we should keep the Scheduling Strategy and Run Schedule close together by shifting the Execution Node below the Concurrent Tasks and Run Schedule.